File: Services/Admin/Admin.Domain/Admin_Review.cs
namespace Admin.Domain;

public class Admin_Review
{
    public int admin_id { get; set; }
    public int review_id { get; set; }
    public static Admin_Review Create(int admin_id,int review_id)
    {
        return new Admin_Review
        {
            admin_id = admin_id,
            review_id = review_id
        };
    }
}

File: Services/DB_Handler/models/AdminReview.py
from sqlalchemy import Column, Integer, ForeignKey
from sqlalchemy.orm import relationship
from .base import Base


class AdminReview(Base):
    __tablename__ = 'admin_review'

    admin_id = Column(Integer, ForeignKey("admins.admin_id"), primary_key=True)
    review_id = Column(Integer, ForeignKey("review.review_id"), primary_key=True)

    admin = relationship("Admins", back_populates="admin_reviews")
    review = relationship("Review", back_populates="admin_reviews")
